### DISCUSSION TITLE
More attributes in dialog.tlk

### DISCUSSION THREAD

**Discussion by Incrementis** [NearInfinityBrowser/NearInfinity#163] [Ideas & Feature Requests]
Hi,

right now I'm working with the dialog.tlk file and when I compare it with the [iesdp documentation](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/tlk_v1.htm) there are some inconsistencies.

### Here is a comparison(see screenshot):
![grafik](https://github.com/NearInfinityBrowser/NearInfinity/assets/17119474/9ec6c099-d7e5-4e55-94ff-e76450f8ec05)
The red areas are attributes not present in Near Infinity, while the green attributes are present in Near Infinity.

### Here's a post on The Gibberlings 3 on why I think it's important (at least to me):
https://www.gibberlings3.net/forums/topic/37053-access-and-heavily-manipulat-dialogtlk/

**As a side-note:** I use Near Infinity as an analysis and debugging tool when coding with WeiDU.

**Comment by Argent77** [comment]
The `dialog.tlk` is not a regular IE game resource which you can manipulate and put into the override folder. Just like the `chitin.key` or BIFF archives it requires special handling.

NI's string table editor hides some of the internals of the dialog.tlk structure since they cannot be modified manually in a meaningful way. It's all handled behind the scenes in the string editor.

If you want to see the internal structure of the dialog.tlk then I'd suggest to open it directly in a hex editor.
